lola-tech/test/hooks/hooks.js

```diff
@@ -4,13 +4,17 @@ let assert = require("chai").assert;
 
 class Hooks extends Page {
   getWebElement(elementName, page) {
+    console.log(elementName);
     elementName = Utils.formatLocator(elementName);
+    console.log(elementName);
```

Comment:
Please remove the console logs

Reply:
I made the requested changes. Please review again.